docs/Configuration.md

```diff
@@ -263,6 +254,39 @@ Custom Annotation names may also be configured using ConfigMap Data Values.
 
 Note: The requirement to label the build with `app.kubernetes.io/name=<app_name>` for the annotated Builds applies.
 
+#### An example workflow for an OpenShift binary build:
+
+```
+NS=mongo-persistent
+NAME=python-binary-build
+
+oc create namespace "${NS}"
+
+oc new-build python --name="${NAME}" --binary=true -n "${NS}"  -l "app.kubernetes.io/name=${NAME}"
+oc start-build "bc/${NAME}" --from-file=./app.py --follow -n "${NS}"
```

Comment:
First step is to ensure `app.py` is in the local folder from which build commands are executed.

Reply:
thanks, fixed